index.jsx

```diff
@@ -33,12 +38,29 @@ export default React.createClass({
       PropTypes.bool,
     ]),
     onChange: PropTypes.func,
-    children: PropTypes.func.isRequired,
+    children: PropTypes.node.isRequired,
+    comp: PropTypes.oneOfType([
```

Comment:
What are you referring to? The existence of the `comp` prop, or the PropType definition? I assume you mean the former:

I've seen it quite often that libs offer a `comp` prop to allow customising the tag for the container. A common use case would be to allow using custom layout containers like `<Box>` or `<Flex>` or <InlineBlock>` or however people wanna call them.

An alternative approach would be that `<RadioGroup>` does not render a tag itself but only it's child. I feel that this is somewhat more inconvenient to use since users might expect that `<RadioGroup>` should be able to render multiple children.